vue upgrade command asks for a plugin and doesn't recognize when it is installed

### Version
3.2.1

### Environment info
```
  System:
    OS: Windows 10
    CPU: (8) x64 Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz
  Binaries:
    Node: Not Found
    Yarn: Not Found
    npm: 6.2.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: 44.17763.1.0
  npmGlobalPackages:
    @vue/cli: Not Found

Note that Node is absolutely installed. "node -v" reports 10.9.0.
```


### Steps to reproduce
1. run "vue upgrade"
2. It reports:

  Command vue upgrade requires a global addon to be installed.
  Please run npm install -g @vue/cli-upgrade and try again.

3. I run the above command, close the terminal, open a new one, and try it again. Same issue.

### What is expected?
For it to attempt to update my CLI.

### What is actually happening?
It reports what is shown in figure 3.

<!-- generated by vue-issues. DO NOT REMOVE -->

If we use `npm i @vue/cli @vue/cli-upgrade`, the `vue upgrade` command will prompt us like
>Command `vue upgrade` requires a global addon to be installed.
  Please run `yarn global add @vue/cli-upgrade` and try again.

Now we open `%AppData%\npm\node_modules\@vue\cli\lib\util\loadCommand.js`, navigate to line 11 `catch (err2)`, add `console.log(err2)`, run ` Vue upgrade`, get the following results:
>Error: Cannot find module '@vue/cli-shared-utils'

However, we can find the `@vue/cli-shared-utils` module in the `%AppData%\Roaming\npm\node_modules\@vue\cli\node_modules\@vue\` directory.

Now let's execute `npm uninstall @vue/cli @vue/cli-upgrade -g` and `yarn global add @vue/cli @vue/cli-upgrade` and `vue upgrade` commands, and get the following result:
>✔ Gathering update information...
  Already up-to-date.

But one thing that puzzles me is:
The expected result could not be obtained yesterday through the following command
```bash
npm i yarn -g
yarn global add @vue/cli @vue/cli-upgrade
vue upgrade
```
I have to use the separate [`yarn` installer](https://www.yarnpkg.com/en/docs/install) to install `yarn` and then execute other commands.
But today there is no problem.

--- a/packages/@vue/cli-upgrade/package.json
+++ b/packages/@vue/cli-upgrade/package.json
@@ -21,6 +21,7 @@
   },
   "homepage": "https://github.com/vuejs/vue-cli/tree/dev/packages/@vue/cli-upgrade#readme",
   "dependencies": {
+    "@vue/cli-shared-utils": "^4.0.0-alpha.0",
     "chalk": "^2.4.1",
     "cli-table": "^0.3.1",
     "execa": "^1.0.0",

